cldfbench_nominalperson_cldf.py

```diff
@@ -1,8 +1,8 @@
+import re
 import pathlib
-import itertools
 
 from cldfbench import Dataset as BaseDataset
-from cldfbench import CLDFWriter as Writer
+from cldfzenodo import Record
```

Comment:
The `cldfzenodo` package provides functionality to retrieve CLDF data deposited on Zenodo.